platform/api/azure/instance.go

```diff
@@ -29,6 +29,8 @@ import (
 	"github.com/flatcar-linux/mantle/util"
 )
 
+var forceDelete = true
```

Comment:
I think it can be safely defined as a constant:
```suggestion
const forceDelete = true
```

Reply:
Tried that, it's not legal to take the address of a constant this way